src/Nordea/Components/DropdownOptions.elm

```diff
@@ -0,0 +1,252 @@
+                    Html.li
+                        [ Events.onClick (option.onCheck (not option.isChecked))
+                        , css
+                            [ maxWidth (pct 100)
+                            , cursor pointer
+                            , hover [ backgroundColor Colors.coolGray ]
+                            , displayFlex
+                            , flexDirection row
+                            , padding (rem 0.5)
+                            , alignItems center
+                            , Css.property "gap" "0.5rem"
+                            ]
+                        ]
+                        [ Checkbox.init option.name (Html.text option.label) option.onCheck
+                            |> Checkbox.withIsChecked option.isChecked
+                            |> Checkbox.withAppearance Checkbox.Simple
+                            |> Checkbox.view []
+                        ]
+
+                dropdownStyles =
+                    Css.batch
+                        [ backgroundColor Colors.white
+                        , borderBottom3 (rem 0.0625) solid Colors.grayMedium
+                        , borderLeft3 (rem 0.0625) solid Colors.grayMedium
+                        , borderRight3 (rem 0.0625) solid Colors.grayMedium
+                        , borderBottomLeftRadius (rem 0.25)
+                        , borderBottomRightRadius (rem 0.25)
+                        , boxSizing borderBox
+                        , padding3 (rem 0.5) (rem 0) (rem 0.0)
+                        ]
+            in
+            Html.div
+                [ css
+                    [ displayFlex
+                    , flexDirection column
+                    , position absolute
+                    , top (pct 100)
+                    , left (rem 0)
+                    , right (rem 0)
+                    , if dropdown.hasFocus then
+                        display block
+
+                      else
+                        display none
+                    ]
+                ]
+                [ Html.ul
+                    [ css
+                        [ overflowY scroll
+                        , maxHeight (rem 16.75)
+                        , dropdownStyles
+                        , listStyle none
+                        ]
+                    ]
+                    (dropdown.options |> List.map viewOption)
+                ]
+    in
+    Html.column attrs
+        [ Html.row [ css [ displayFlex, justifyContent spaceBetween, marginBottom (rem 0.2) ] ]
+            [ Common.viewLabel { label = dropdown.label, isError = False } []
+            , dropdown.requirednessHint |> Html.viewMaybe RequirednessHint.view
+            ]
+        , Html.div
+            [ class "nordea-select"
+            , Events.on "focusout" (Decode.succeed (dropdown.onFocus False))
+            , Events.on "focusin" (Decode.succeed (dropdown.onFocus True))
+            , css [ position relative ]
+            , tabindex -1
```

Comment:
Had to add `tabindex -1` to be able to receive focus🤔 Is there another way to do it?

Reply:
I think this is the correct way to add focus support to divs (and other elements) according to the spec